tests/sampletester/ProcessHelper.cs

```diff
@@ -89,7 +95,7 @@ public static void BuildSolution (string solution, string platform, string confi
 
 		foreach (var sln in solutions) {
 			nuget_args [1] = sln; // replacing here
-			AssertRunProcess ("nuget", nuget_args.ToArray (), TimeSpan.FromMinutes (2), Configuration.SampleRootDirectory, environment_variables, "nuget restore");
+			AssertRunProcess ("nuget", nuget_args.ToArray (), TimeSpan.FromMinutes (5), Configuration.SampleRootDirectory, environment_variables, "nuget restore");
```

Comment:
👍 I though that the 2 minutes was an internal timeout of `nuget` :)
https://github.com/xamarin/maccore/issues/1616